.github/workflows/build.py

```diff
@@ -67,18 +67,19 @@ def delete_docker_images():
 def make_builds(benchmarks, fuzzer):
     """Use make to build each target in |build_targets|."""
     make_targets = get_make_targets(benchmarks, fuzzer)
-    success = True
     for pull_target, build_target in make_targets:
         # Pull target first.
         subprocess.run(['make', '-j', pull_target], check=False)
 
         # Then build.
-        result = subprocess.run(['make', build_target], check=False)
+        print('Building', build_target)
+        build_command = ['make', '-j', build_target]
+        result = subprocess.run(build_command, check=False)
         if not result.returncode == 0:
-            success = False
```

Comment:
Could be simplified to "if result.return_code:"

Reply:
I think this simplification makes it harder to understand what is going on.